pkg/grapicmd/context.go

```diff
@@ -102,3 +107,24 @@ func (c *Ctx) loadConfig() error {
 func (c *Ctx) IsInsideApp() bool {
 	return c.insideApp
 }
+
+// CtxSet is a provider set that includes modules contained in Ctx.
+var CtxSet = wire.NewSet(
+	ProvideFS,
+	ProvideViper,
+	ProvideExecer,
+	ProvideIO,
+	ProvideRootDir,
+	ProvideConfig,
+	ProvideBuildConfig,
+	ProvideProtocConfig,
+)
+
+func ProvideFS(c *Ctx) afero.Fs                 { return c.FS }
+func ProvideViper(c *Ctx) *viper.Viper          { return c.Viper }
+func ProvideExecer(c *Ctx) exec.Interface       { return c.Execer }
+func ProvideIO(c *Ctx) *cli.IO                  { return c.IO }
+func ProvideRootDir(c *Ctx) cli.RootDir         { return c.RootDir }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideRootDir should have comment or be unexported